src/main/java/rs/hybridit/service/BookInstanceService.java

```diff
@@ -0,0 +1,13 @@
+package rs.hybridit.service;
+
+import rs.hybridit.model.BookInstance;
+
+import java.util.List;
+
+public interface BookInstanceService {
+    public BookInstance getOne(long id);
+    public List<BookInstance> getAll();
+    public BookInstance create(BookInstance book);
+    public BookInstance save(BookInstance book);
+    public void delete(BookInstance book);
```

Comment:
a lot of repetitive methods, you can refactor this and make one abstract interface, but this may be too advanced
@aronsrederhybrid what do you think about this?